backend/api/Areas/Tools/Helpers/ImportPropertiesHelper.cs

```diff
@@ -297,6 +297,13 @@ public IEnumerable<Model.ImportPropertyModel> AddUpdateProperties(IEnumerable<Mo
                             property.Added = true;
                             property.Updated = false;
                         }
+                        else if (isBuildingExisting.Count() >= 2)
+                        {
+                            // there seems to be more than one building with the same name and address.....need to determine which one to update?
+                            property.Added = false;
+                            property.Updated = false;
+                            property.Error = isBuildingExisting.Count() + " buildings were found with the same name. Couldn't tell which one to update.";
+                        }
```

Comment:
Good idea. I wonder if we should just throw an exception here though, as the catch already handles setting Added and Updated to false and setting the error message.

```cs
throw new Exception(isBuildingExisting.Count() + " buildings were found with the same name. Couldn't tell which one to update.");
```

Reply:
Also because if we don't throw, I think it will still attempt the update at the bottom of this `try` block.